site/content/en/docs/faq/_index.md

```diff
@@ -18,6 +18,20 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How to create an multiple clusters using minikube ?
```

Comment:
I would move away from the word `multiple` so people don't get confused with multinode.

Maybe: `How can I create additional clusters?`